source/atlas-cli-deploy-docker.txt

```diff
@@ -0,0 +1,148 @@
+
+.. _atlas-cli-deploy-docker-setup:
+
+Create a Local Atlas Deployment in Docker
+-----------------------------------------
+
+Use the ``atlas deployments`` command to create a local |service|
+deployment in Docker.
+      
+.. procedure::
+   :style: normal
+
+   .. step:: Install and start Docker.
+
+      To learn more, see the `Docker documentation <https://docs.docker.com/desktop/install/mac-install/>`__.
+
+   .. step:: Pull down the latest ``mongodb/atlas`` Docker image.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         docker pull mongodb/atlas:latest
+
+   .. step:: Run the Docker image in bash mode.
+
+      To learn more, see :ref:`atlas-cli-docker`.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         docker run -p 27777:27017 --privileged -it mongodb/atlas bash
+
+   .. step:: Create a local |service| deployment.
+
+      The following command should return a connection string.
+
+      **Example:**
+
+      .. code-block::
+
+         atlas deployments setup --bindIpAll --username root --password root --type local --force
+
+   .. step:: Connect to the local |service| deployment.
+
+      To connect to the local |service| deployment from the host (not 
+      container), copy and paste the following command, and replace 
+      the ``{connection_string}`` variable with your connection string.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         mongosh {connection_string} --username root --password root
+
+
+Create a Local Atlas Deployment in Docker Compose
+-------------------------------------------------
+
+Use the ``atlas deployments`` command to create a local |service|
+deployment in Docker Compose.
+      
+.. procedure::
+   :style: normal
+
+   .. step:: Install Docker Compose.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         brew install docker-compose
+
+   .. step:: Create a docker-compose.yaml file.
+
+      **Example:**
+
+      .. code-block:: sh
+         
+         services:
+           mongo:
+             image: mongodb/atlas
+             privileged: true
+             command: |
+               /bin/bash -c "atlas deployments setup --type local --port 27778 --bindIpAll --username root --password root --force && tail -f /dev/null"
+             volumes:
+               - /var/run/docker.sock:/var/run/docker.sock
+             ports:
+               - 27778:27778
+
+   .. step:: Run Docker Compose.
+
+      The following command should return a connection string.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         docker-compose up
+    
+   .. step:: Connect to the local |service| deployment.
+
+      To connect to the local |service| deployment from the host (not 
+      container), copy and paste the following command, and replace 
+      the ``{connection_string}`` variable with your connection string.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         mongosh {connection_string} --username root --password root
+
```

Comment:
It might be helpful for users to provide an example of the connection string. Here and in the section above as well.